esmvaltool/preprocessor/_multimodel.py

```diff
@@ -73,7 +73,10 @@ def _compute_statistic(datas, name):
         # get all NOT fully masked data - u_data
         # datas is per time point
         # so we can safely NOT compute stats for single points
-        u_datas = [data for data in datas if not np.all(data.mask)]
+        if datas.ndim == 1:
```

Comment:
@valeriupredoi Can you comment on this? Does it work as intended?

Reply:
Just to add some context, this section was added because I ran the `time_average` preprocessor followed by the multi model mean. The time average preprocessor produces a 1D array, which was not among the range of possibilities considered by the multimodel preprocessor.

Reply:
Time operations are still not part of the preprocessor, although a draft module `_time_area.py` is available. These kind of operations should anyway be performed at the end of the preprocessing chain, i.e. after level selection, regrdding, fill value masking and multi-model-stats. This is the way the preprocessing chain is currently designed, since a customizable order of operations is not yet available (altough planned). 

Hence my suggestion here would be to move the `time_average` at the end of the chain.

Reply:
this should work OK, I've just tested with this branch as well and basic preprocessing works a-ok

Reply:
@mattiarighi, I've added several functions to the preprocessor, and I think I've added all of them into the _time_area.py module. However, very few of them are time manipulations. Would it be better to split this into several files with different categories, ie, time, area, volume?